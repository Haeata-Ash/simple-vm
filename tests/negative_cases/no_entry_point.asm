FUNC LABEL 7
	PRINT VAL 7
	RET
