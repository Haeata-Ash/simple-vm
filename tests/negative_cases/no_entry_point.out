No entry point in program
